velox/exec/Operator.h

```diff
@@ -107,6 +107,7 @@ struct OperatorStats {
   CpuWallTiming finishTiming;
 
   MemoryStats memoryStats;
+  uint64_t spilledBytes{0};
```

Comment:
Would you add a small comment? Would it make sense to also track number of spilled rows?